recipes/sqlite_zstd_vfs/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "sqlite_zstd_vfs" %}
+{% set version = "2024.01.13" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/mlin/sqlite_zstd_vfs
```

Comment:
Please use an archive from this instead of cloning via git. Or do you need some git metadata here?

Reply:
Changed it to an archive in 6c997ea